nautobot/docs/development/getting-started.md

```diff
@@ -318,6 +318,13 @@ It may not always be convenient to enter into the virtual shell just to run prog
 $ poetry run mkdocs serve
 ```
 
+* Install verify that you have the proper dependencies installed and are in the virtual environment via Poetry. This also ensures that you have the proper mkdocs themes installed.
+
+```bash
+poetry shell
+poetry install
+```
+
```

Comment:
Isn't this still redundant with the immediately preceding section of the docs?

Reply:
Updating the order to provide better context.

Reply:
Personally I still feel this is redundant with the information under `#### Creating a Python Virtual Environment` that explicitly instructs you to run `poetry install` and `poetry shell`, but I won't block this.